packages/insured-bridge-relayer/src/Relayer.ts

```diff
@@ -138,12 +132,120 @@ export class Relayer {
         );
       }
     }
+
+    return;
+  }
+
+  async checkForPendingRelaysAndDispute(): Promise<undefined> {
```

Comment:
```suggestion
  async checkForPendingRelaysAndDispute(): Promise<void> {
```

nit: prefer void for return values for functions that don't return anything.

Reply:
done